ycms/cms/views/zentral.py

```diff
@@ -0,0 +1,96 @@
+from ..forms import WardForm
+from ..models import User, Ward
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+
+    template_name = "zentral.html"
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        This function returns a list of all wards,
+        as well as a form for creating a new one(not implemented yet)
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+
+        """
+        # ward list which acts as a iterable object
+        total_wards = Ward.objects.all()
+        # total number of wards
+        total_wards_count = Ward.objects.all().count()
+        # total number of beds
+        total_beds_count = total_wards_count * 28
+        # total occupied beds
+        total_occupied_beds = 0
+        total_available_beds = 0
+        for ward in total_wards:
+            total_occupied_beds += ward.occupied_beds
+            total_available_beds += ward.available_beds
+        # total available beds
+        # total_available_beds =  #looping through each ward and multiply with Ward.available_beds method
+        # total number of doctors
+        total_doctors_count = User.objects.filter(job_type=job_types.DOCTOR).count()
+        # total number of nurses
+        total_nurses_count = User.objects.filter(job_type=job_types.NURSE).count()
+
+        # context = {"wards list":wards_list}
+        ward_form = WardForm()
+
+        return render(
+            request,
+            self.template_name,
+            {
+                "wards": total_wards,
+                "wardscount": total_wards_count,
+                "beds": total_beds_count,
+                "doctors": total_doctors_count,
+                "nurses": total_nurses_count,
+                "availablebeds": total_available_beds,
+                "occupiedbeds": total_occupied_beds,
+                "wardform": ward_form,
+            },
+        )
+
+    def post(self, request, *args, **kwargs):
+        r"""
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: Redirect to list of wards
+        :rtype: ~django.http.HttpResponseRedirect
+        """
+        ward_form = WardForm(
+            data=request.POST, additional_instance_attributes={"creator": request.user}
+        )
+        if not ward_form.is_valid():
+            ward_form.add_error_messages(request)
+            return redirect("cms:protected:zentral")
+        ward = ward_form.save()
+        messages.success(
+            request, _('Addition of new ward "{}" successful!.').format(ward.name)
```

Comment:
```suggestion
            request, _('Addition of new ward "{}" successful!').format(ward.name)
```
(THen you will also have to do this, otherwise the translation will not recognize the changed string)